setup.py

```diff
@@ -146,7 +148,9 @@ def get_extras_require() -> Dict[str, List[str]]:
             "scikit-learn>=0.24.2",
             "scikit-optimize",
             "xgboost",
-            "keras ; python_version<'3.9'",
```

Comment:
Just curious but could you comment on the removal of the Python version constraint?

Reply:
I removed It accidentally. I'll revert the condition. Thanks!